src/main/java/org/prebid/server/hooks/execution/HookStageExecutor.java

```diff
@@ -254,6 +258,22 @@ public Future<HookStageExecutionResult<AuctionResponsePayload>> executeAuctionRe
                 .execute();
     }
 
+    public Future<HookStageExecutionResult<ExitpointPayload>> executeExitpointStage(MultiMap responseHeaders,
+                                                                                    String responseBody,
+                                                                                    AuctionContext auctionContext) {
+
+        final Account account = ObjectUtils.defaultIfNull(auctionContext.getAccount(), EMPTY_ACCOUNT);
+        final HookExecutionContext context = auctionContext.getHookExecutionContext();
+
+        final Endpoint endpoint = context.getEndpoint();
+
+        return stageExecutor(StageWithHookType.EXITPOINT, ENTITY_HTTP_RESPONSE, context, account, endpoint)
+                .withInitialPayload(ExitpointPayloadImpl.of(responseHeaders, responseBody))
```

Comment:
Perhaps we need to wrap `responseHeaders` in an unmodifiable implementation. @And1sS what do you think?

Reply:
do you mean we want to make module to recreate the full map instead of updating what is passed? it will be an additional step for the module creators